test/name_bazaar/run_tests.cljs

```diff
@@ -4,9 +4,22 @@
     [name-bazaar.smart-contract-tests]
     [name-bazaar.smart-contract-ext-tests]
     [name-bazaar.smart-contract-altering-tests]
+    [clojure.string :as str]
     [cljs.test :refer-macros [run-tests]]))
 
-(nodejs/enable-util-print!)
+(defn enable-util-print! []
```

Comment:
No, didn't mean like this. I'm 95% sure the error is printed on this line https://github.com/wambat/name-bazaar/blob/master/src/district0x/server/effects.cljs#L124. If `:log-contract-calls?` is false, shouldn't print event errors